packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -109,6 +109,65 @@ For password-only authentication, you need to update the password authenticator
 1. In the **Edit Rule** dialog box, select **Not required** in the **AND Additional verification is** section.
 1. Click **Update Rule**.
 
+### Set up your Okta org for a password-optional use case
+
+Set up your Okta your org to enable authentication without a password using the followings steps:
```

Comment:
```suggestion
Set up your Okta org to enable authentication without a password using the followings steps:
```

Reply:
Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435